lib/engine/game/base.rb

```diff
@@ -650,7 +650,19 @@ def end_game!
       end
 
       def revenue_for(route, stops)
-        stops.sum { |stop| stop.route_revenue(route.phase, route.train) }
+        stops.sum { |stop| revenue_for_stop(route, stop) }
+      end
+
+      def revenue_for_stop(route, stop)
+        distance = route.train.distance
+        multiplier = 1
```

Comment:
There might be a cleaner way to write this in Ruby. I come from the C/JS/PHP world and am still learning Ruby syntax.